src/report/business-reports/aging.scm

```diff
@@ -409,8 +510,8 @@ totals to report currency.")
     (define oldintervalreversed (reverse (make-interval-list to-date)))		
   (reverse (cons dayforcurrent oldintervalreversed)))
 
-(define (aging-renderer report-obj reportname account reverse?)
-
+(define (aging-renderer report-obj reportname account reverse? receivable)
+	
```

Comment:
If you are working with an internal option, you don't need to add the receivable parameter here either. You can query it later whenever you need it using the "op-value" function defined a bit further in the code.